examples/demos/selectable.js

```diff
@@ -14,6 +14,7 @@ let Selectable = React.createClass({
           selectable
           events={events}
           defaultView='week'
+          defaultTimeScroll={new Date(1970, 1, 1, 17)}
```

Comment:
this isn't a better name I don't think. the "default" version of props also have a non default counterpart (this does not). Default props also imply that the prop is only set once which isn't true in this case. can we use scrollToTime instead?